packages/react/src/components/Button/Button.module.css

```diff
@@ -25,7 +25,7 @@
   overflow: visible;
```

Comment:
Do we still need this?

Reply:
Hmm, its a bit weird indeed. I can't think of why we should have it anymore. Should anyways be a separate PR if we do figure out to change this :)